custom_components/powercalc/data/lifx/LIFX A19 Night Vision/README.md

```diff
@@ -0,0 +1,3 @@
+Notes:
+
+- TP-link kp125 was having inconsistent readings towards the 0.52 watt and lower range, so I manually smoothed out some of the outliers.
```

Comment:
You may add this to `measure_description` in model.json. Maybe in the future I will generate more UI pages for the light library, than we can also display this extra information.

Reply:
Great idea, changed.